Source/WebKit/WebProcess/Plugins/PDF/PDFPluginBase.h

```diff
@@ -50,6 +50,7 @@
 #include <wtf/TypeTraits.h>
 #include <wtf/WeakPtr.h>
 
+OBJC_CLASS NSData;
```

Comment:
Do you need `OBJC_CLASS NSString;` as well?

Reply:
I do, I'll add it so we don't implicitly depend on a transitive `Foundation/Foundation.h` import.